### ISSUE TITLE
[Feature] support The Arrange, Act, and Assert (AAA) Pattern

### ISSUE THREAD

**Issue by 304NotModified** [mattwhitfield/unittestgenerator#45]
Would be great if the comments for AAA are generated. (`// Arrange`, `// Act`, `// Assert`)

e.g.

```c#
// Arrange
var repository = Substitute.For<IClientRepository>();
var client = new Client(repository);

// Act
client.Save();

// Assert
mock.Received.SomeMethod();
```

**Comment by mattwhitfield** [comment]
Good idea, I'll make it customisable so you can configure what you want for each of the comments...

**Comment by mattwhitfield** [comment]
I've got an initial implementation - and I have re-generated the docs here: https://github.com/mattwhitfield/unittestgenerator/tree/feature/SectionComments/docs/examples

I need to add the ability to configure it so that you can change the text or even turn the comments off, but it's a start 👍

**Comment by 304NotModified** [comment]
Very nice!

**Comment by mattwhitfield** [comment]
Available in 0.96 🙂

If you want to change the comments that are emitted, they can be changed via Tools -> Options -> Unitverse -> Generation Options. Or in a `.unitTestGeneratorConfig` file, of course. 👍

**Comment by 304NotModified** [comment]
Thanks!
